### PR TITLE
py-httplib2:Version update

### PR DESCRIPTION
**Pull request by dorton21**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-httplib2/package.py [hunk 1]
```diff
@@ -12,6 +12,8 @@ class PyHttplib2(PythonPackage):
     homepage = "https://github.com/httplib2/httplib2"
     pypi = "httplib2/httplib2-0.13.1.tar.gz"
 
+    version('0.18.0', sha256='b0e1f3ed76c97380fe2485bc47f25235453b40ef33ca5921bb2897e257a49c4c')
     version('0.13.1', sha256='6901c8c0ffcf721f9ce270ad86da37bc2b4d32b8802d4a9cec38274898a64044')
 
     depends_on('py-setuptools', type='build')
+    depends_on('py-pyparsing@2.4.2:2.999.999', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-httplib2/package.py]: Where do you see this dependency?

Comment (dorton21) [reply var/spack/repos/builtin/packages/py-httplib2/package.py]: https://github.com/httplib2/httplib2/blob/master/requirements.txt

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-httplib2/package.py]: Never trust `requirements.txt`: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#python-dependencies

That file isn't even in the 0.13.1 release.

Comment (dorton21) [reply var/spack/repos/builtin/packages/py-httplib2/package.py]: Good to know. Should I just remove it then?

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-httplib2/package.py]: Yes, probably

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**